papers/Core_Team.md

```diff
@@ -0,0 +1,256 @@
+Active Core Team:
```

Comment:
This "Core Team" should be reproducible, and at this point you should probably say how you obtained this list. I can foresee some problems with someone wanting, or not wanting, to be a core team member.